.github/workflows/ci.yml

```
@@ -51,7 +51,8 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macOS-latest]
+        # os: [ubuntu-latest, windows-latest, macOS-latest]
+        os: [ubuntu-latest, macOS-latest]
```

Let's try reverting and using `language_std(&mut base_config, "c++20")` ?

```suggestion
        os: [ubuntu-latest, windows-latest, macOS-latest]
```